vrf.go

```diff
@@ -23,18 +28,37 @@ func getDomainFromAddress(address string) (string, error) {
 	return address[at+1:], nil
 }
 
-func isDeliverable(host string, address string) (bool, error) {
+func isDeliverable(host string, address string, timeout ...time.Duration) (bool, error) {
```

Comment:
Why a variadic arg instead of just accepting `nil`?

Reply:
I felt that overloading the function is more flexible if we want to make this an optional argument. Also, I suspected that the function may already be in use somewhere, and this would make it backward compatible?